packages/framework/src/Resources/views/Admin/Content/Currency/currencySettings.html.twig

```diff
@@ -1,12 +1,14 @@
 {{ form_start(form) }}
-    <div class="form-group">
-        <div class="form-group__title">
-            {{ 'Settings'|trans }}
-        </div>
-        {{ form_row(form.defaultCurrency, { label: 'Default administration currency'|trans, icon: { title: 'When changing the default currency there is no recalculation of prices'|trans } }) }}
-        {% for item in form.domainDefaultCurrencies %}
-            {{ form_row(item, { label: 'Basic frontend currency %domainName%'|trans({'%domainName%': domainNames[item.vars.name]})}) }}
-        {% endfor %}
+        <div class="form-group">
+            <div class="form-group__title">
+                {{ 'Settings'|trans }}
+            </div>
+            <div>
```

Comment:
Why are you adding the `div`?

Reply:
Same as higher. This parent makes child - first child and last child so it gets border radius

Reply:
:+1: